Palette.js

```diff
@@ -1,6 +1,6 @@
 class Palette {
   constructor() {
-    this.id = Date.now();
+    this.id = Date.now().toString()
     this.colors = [
       new Color(),
       new Color(),
```

Comment:
this.id was returning a `number` but our id for the `trash icon` is using that same id except in a `string.' I updated the color id to generate like how it was before but return it in the string format at the end.

Reply:
Awesome. That should then work the locks and the delete function at the same time without messing with each other.